schemas/entity-styles.json

```diff
@@ -0,0 +1,44 @@
+{
```

Comment:
These schemas are separate so they can be hosted with the spec, and you can use `"$ref": "https://identity.foundation/wallet-rendering/schemas/entity-styles.json"` in other schemas